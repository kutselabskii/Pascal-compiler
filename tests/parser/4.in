program ABC;
const a = 5;
      b = 4.e12;
      c = #100;
      d = 'hello';
begin
end
.