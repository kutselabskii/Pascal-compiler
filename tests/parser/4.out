|__ program
    |-- abc //Program_heading
    |__ //Block
        |-- //Declaration part
        |   |-- const
        |   |   |-- a
        |   |   |   |__ 5
        |   |   |__ b
        |   |       |__ 4e+12
        |   |   |__ c
        |   |       |__ d
        |   |   |__ d
        |   |       |__ hello
        |__ //Statement part
            |__ //Statement sequence
